modules/yarn.lock

```diff
@@ -4216,71 +4216,82 @@ clay-icon@2.21.4, clay-icon@^2.21.4:
     metal-state "^2.16.0"
     metal-web-component "^2.16.0"
 
-clay-label@2.21.4, clay-label@^2.21.4:
-  version "2.21.4"
-  resolved "https://registry.yarnpkg.com/clay-label/-/clay-label-2.21.4.tgz#5495aff28caa9dcff3d63bafc611b5ec60e6e0a0"
-  integrity sha512-DCgXL4pQTfqlntpNacjw0njGPo3jHLEn5+mCJHfYgmgZJ/SNBDtOD7syFmuo60vY/KfXSABOw6NNnUKU1UbZBA==
+clay-icon@2.22.0, clay-icon@^2.21.4, clay-icon@^2.22.0:
```

Comment:
Wonder where's this fixed dependency coming from... my bet, commerce is sneaking a lot of packages still outside of SF that weren't updated maybe?

Reply:
There are other packages also duplicated. Diff in addition/deletion in `yarn.lock` usually gives a hint about this ;)

Reply:
> commerce is sneaking a lot of packages still outside of SF that weren't updated maybe

Yeah you are right, I forgot they have to be manually updated due to SF suppression. I'll take a look!